alphacsc/utils/signal.py

```diff
@@ -11,3 +11,40 @@ def fast_hilbert(array):
 def next_power2(num):
     """Compute the smallest power of 2 >= to num.(float -> int)"""
     return 2 ** int(np.ceil(np.log2(num)))
+
+
+def split_signal(X, n_splits=1, apply_window=True):
```

Comment:
Nice !

Reply:
Ideally, we should call this function in `alphacsc.datasets.somato.load_data`.

Reply:
Ah, just saw this. Shouldn't there be a way to do an "inverse transform" on this to recover the actual Z ?

Reply:
No because if you compute `z_hat_split = cfl.transform(X_split)`, the inverse transform for this won't be equivalent to `z_hat` because of border effect. 

Also, this transform is lossy as it resize all split to `n_times // n_splits` so I am not sure it is worth implementing the inverse transform as it won't result in `X = inv_spkit_signal(split_signal(X))`

Ideally, we want to stop using this and use dicodile instead to provide multiprocessing computations.

Reply:
Ah I see. I was going to say that this is not needed because we do `cdl.transform(X)` but I presume it's not mathematically equivalent to what you would get if you did not have the border effect. +1 for using dicodile then :-)